### ISSUE TITLE
mupdf leak incorrectly marked as flaky

### ISSUE THREAD

**Issue by titanous** [google/oss-fuzz#1095]
https://oss-fuzz.com/v2/testcase-detail/5716457297018880

This test case is marked as flaky but it appears to reproduce consistently locally and `-minimize` works fine too.

I'm not sure if this is a bug or expected?

**Comment by titanous** [comment]
Hmm, actually `-minimize` switches from a leak to an OOM eventually.

**Comment by inferno-chromium** [comment]
Reuploaded testcase as https://oss-fuzz.com/v2/testcase-detail/5872996909580288 to see if results change. Sometimes test can be flaky if they jump stack too much. Our bots are 1-core gce vms so could be flaking with the oom issue you cited.

**Comment by inferno-chromium** [comment]
Ok, reuploaded testcase works. But looked at logs more.

When crash was initially found during fuzzing, it crashed on a different leak stacktrace. Then afterwards when we were just testing only the testcase itself, we found a different stacktrace. So, we marked it as flaky. We can give up old stack but then we dont really know if this is even related to the original one found during fuzzing. It is a hard choice.

Detected a crash with an unrelated state: Expected(do_scavenging_malloc cmsOpenIOhandlerFromMem cmsOpenProfileFromMemTHR ), Found(do_scavenging_malloc fz_calloc fz_new_device_of_size ).
